index.html

```diff
@@ -9,7 +9,7 @@
 </head>
 <body>
 <section class="Home"></section>    
-<header class="home-header">
+<header class="mob-home-header">
     <div class="name logo">
         <div class="name-text" ><a href="" id="logo">Sungabanja Thawethe</a></div>
     </div>
```

Comment:
[OPTIONAL] You can use your short name here. For example, my full name is `Rao Muhammad Akif Tufail`. But it does look quite expanded. So I use a short one like `Rao Akif`. This is an optional suggestion. If you feel OK with your short name, that's great. By the way, I have also seen some guys, making a text logo, that takes a square or a circular shape, which also looks fine. Think about what feels good for design.